server/src/instant/superadmin/routes.clj

```diff
@@ -79,38 +86,60 @@
        </p>")}))
 
 (defn app-transfer-send-invite-post [req]
-  (let [{user-id :id} (req->superadmin-user! req)
-        id (ex/get-param! req [:params :app_id] uuid-util/coerce)
-        {app-id :id} (app-model/get-by-id-and-creator! {:user-id user-id
-                                                        :app-id id})
+  (let [{:keys [app user]} (req->superadmin-user-and-app! req)
         invitee-email (ex/get-param! req [:body :dest_email] email/coerce)
+        {app-id :id} app
+        {user-id :id} user
         {invite-id :id} (instant-app-member-invites-model/create!
                          {:app-id app-id
                           :inviter-id user-id
                           :email invitee-email
                           :role "creator"})]
-    (instant-app-member-invites-model/create!
-     {:app-id app-id
-      :inviter-id user-id
-      :email invitee-email
-      :role "creator"})
     (postmark/send!
-     (transfer-app-invite-email
-      {:inviter-id user-id
-       :invitee-email invitee-email
-       :app-id app-id}))
+     (transfer-app-invite-email user app invitee-email))
     (response/ok {:id invite-id})))
 
 (defn app-transfer-revoke-post [req]
-  (let [{user-id :id} (req->superadmin-user! req)
+  (let [{{user-id :id} :user {app-id :id} :app} (req->superadmin-user-and-app! req)
         dest-email (ex/get-param! req [:body :dest_email] email/coerce)
         rejected-count (count (instant-app-member-invites-model/reject-by-email-and-role
                                {:inviter-id user-id
+                                :app-id app-id
                                 :invitee-email dest-email
                                 :role "creator"}))]
 
     (response/ok {:count rejected-count})))
 
+;; --------- 
+;; Rules 
+
+(defn app-rules-get [req]
+  (let [{{app-id :id} :app} (req->superadmin-user-and-app! req)
+        {:keys [code]} (rule-model/get-by-app-id aurora/conn-pool {:app-id app-id})]
+    (response/ok {:perms code})))
+
+(defn app-rules-post [req]
+  (let [{{app-id :id} :app} (req->superadmin-user-and-app! req)
+        code (ex/get-param! req [:body :code] w/stringify-keys)]
+    (ex/assert-valid! :rule code (rule-model/validation-errors code))
+    (response/ok {:rules (rule-model/put! {:app-id app-id
+                                           :code code})})))
+
+;; --------- 
+;; Schema 
+
+(defn app-schema-plan-post [req]
```

Comment:
alternatively, this _could_ have lived in the admin SDK too. But, I thought it would be a nicer dev experience for platform, if they _only_ had to use one -- the personal access token. I went ahead and wrote those here for user convenience.